apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -67,8 +73,15 @@ export const blocks = {
   },
 
   // Adds a toggle button field to a block. Requires other inputs to already exist.
-  appendMiniToolboxToggle(miniToolboxBlocks, flyoutToggleButton) {
+  appendMiniToolboxToggle(
+    miniToolboxBlocks,
+    flyoutToggleButton,
+    renderingInFunctionEditor = false
+  ) {
+    // In the function editor, this call prevents a dummy input from being used as a
+    // row separator between the function definition in the mini-toolbox.
     this.setInputsInline(true);
```

Comment:
FYI @mikeharv @molly-moen I realized that this line wasn't what broke the row separators; it was that the renderer wasn't being passed to the modal workspace correctly, so we still weren't using the Thrasos renderer with the row-separator extension.